types.go

```diff
@@ -10,6 +10,11 @@ import (
 // that will be initialized in the kernel.
 type MapType uint32
 
+// Max returns the
+func (_ MapType) Max() MapType {
+	return maxMapType
```

Comment:
I've been thinking about this; we should probably make this return `maxMapType-1`, since the `maxMapType` value points past what's actually supported.